Scripts/parameters/freight/metsat.json

```diff
@@ -26,20 +25,21 @@
             "impedance": {},
             "log": {
                 "attraction_size": 1.0,
-                "logsum": 0.6096
+                "logsum": 0.2307
             },
             "attraction_size": {
-                "att_metsat": 1.0
+                "att_metsat": 1
             }
         }
     },
     "mode_choice": {
         "truck": {
-            "constant": 0.0,
+            "constant": 0,
             "generation": {},
             "attraction": {},
             "impedance": {
-                "cost": -0.7224
+                "cost_log": -6.524,
+                "cost": -0.2782
             },
             "log": {},
```

Comment:
We have the separate "log" structure for this, where you can put "cost". It will then logarithmize it and then apply the parameter.

Reply:
I see, that's what the log section is for. I'll update this particular model.